site/public/mjs/sql-toolbox.js

```diff
@@ -36,7 +36,7 @@ async function runSqlQuery() {
         if (data.length === 0) {
```

Comment:
Here as well, it's reporting missing branches, but as far as I can tell both branches are covered.